tests/test_io_converters.py

```diff
@@ -0,0 +1,109 @@
+import itertools
+from pathlib import Path
+import pytest
+import subprocess
+import meeteval
+
+example_files = (Path(__file__).parent.parent / 'example_files').absolute()
+
+test_files = {
+    'ctm': 'hyp1.ctm',
+    'stm': 'hyp.stm',
+    'rttm': 'hyp.rttm',
+    'seglst': 'hyp.seglst.json',
+}
+
+def run(cmd):
+    cp = subprocess.run(
+        cmd,
+        shell=True,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+        check=False,
+        universal_newlines=True,
+        cwd=example_files,
+        executable='bash',  # echo "<(cat hyp.stm)" requires bash not sh.
+    )
+
+    if cp.returncode == 0:
+        return cp
+    else:
+        if not isinstance(cmd, str):
+            import shlex
+            cmd = shlex.join(cmd)
+        raise Exception(
+            f'$ {cmd}'
+            f'\n\nreturncode: {cp.returncode}'
+            f'\n\nstdout:\n{cp.stdout}'
+            f'\n\nstderr:\n{cp.stderr}'
+        )
+
+
+@pytest.fixture
+def output_folder():
+    output_folder = Path('converted').absolute()
```

Comment:
How about using `tempfile.TemporaryDirectory`?

Reply:
pytest has a build-in feature for temporary directories: https://docs.pytest.org/en/4.6.x/tmpdir.html

I updated the code to use that